### ISSUE TITLE
Issues after deleting group avatar

### ISSUE THREAD

**Issue by aNNiMON** [telegramdesktop/tdesktop#5914]
### Steps to reproduce
1. Go to group settings and upload pic1, pic2 and pic3
  ![image](https://user-images.githubusercontent.com/1321838/55911540-919cf100-5be9-11e9-9f40-a230c0ca4480.png)
2. Delete pic3
  ![image](https://user-images.githubusercontent.com/1321838/55911471-6e724180-5be9-11e9-9fff-b1eda1c2478c.png)

### Expected behaviour
After deleting pic3, the group picture should be set to pic2

### Actual behaviour
After deleting pic3 there is no group picture at all:
![image](https://user-images.githubusercontent.com/1321838/55911516-82b63e80-5be9-11e9-8a1b-f2363d119c65.png)

But if we click on the image above service message
![image](https://user-images.githubusercontent.com/1321838/55911917-4b945d00-5bea-11e9-87ec-f696d20e15b5.png)
the list of all group avatars will appear
![image](https://user-images.githubusercontent.com/1321838/55911540-919cf100-5be9-11e9-9f40-a230c0ca4480.png)

Now set pic4 as the group avatar
![image](https://user-images.githubusercontent.com/1321838/55912916-1557dd00-5bec-11e9-9e8e-69b68f9c8a81.png)
All previous group avatars will appear (including removed pic3)

----

One more thing. If we try to remove latest pic (now it's pic4), we get this message:
![image](https://user-images.githubusercontent.com/1321838/55913219-c9f1fe80-5bec-11e9-818c-2ae4b45b45ea.png)

But if we try to remove old pics, we get another message
![image](https://user-images.githubusercontent.com/1321838/55913201-bc3c7900-5bec-11e9-9e27-0c9d2b4d681e.png)

----

#5279 may be related to this, but it's different issue

### Configuration
**Operating system:** Windows 7

**Version of Telegram Desktop:** 1.6.5 beta

**Used theme**: Any

**Comment by aNNiMON** [comment]
Still an issue in 2.4.3 :cry:

**Comment by aNNiMON** [comment]
2.7.1, stiil an issue
Almost 2 years :tada:

**Comment by john-preston** [comment]
@aNNiMON About the "One more thing. If we try to remove latest pic (now it's pic4), we get this message:"

The list of group profile photos is really a list of messages about changing the group photo. So deleting the old ones is about deleting the corresponding messages. The last one is deleted from the group, clearing the group photo.

**Comment by aNNiMON** [comment]
Issue still exists

**Comment by aNNiMON** [comment]
Issue still exist in 3.6.1. I'm waiting for 3.6.2+ to check if 1bd4595bba8f73ee78e31670f2f42a747d620f7a fixes this.

**Comment by aNNiMON** [comment]
Still reproducible in 3.6.3

**Comment by aNNiMON** [comment]
Issue still exists

**Comment by Nguyenpham89** [comment]
Tơi sẽ nâng cấp

Tải Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Từ: github-actions[bot] ***@***.***>
Đã gửi: Wednesday, April 12, 2023 8:53:19 AM
Đến: telegramdesktop/tdesktop ***@***.***>
Cc: Subscribed ***@***.***>
Chủ đề: Re: [telegramdesktop/tdesktop] Issues after deleting group avatar (#5914)


Hey there!

This issue was inactive for a long time and will be automatically closed in 30 days if there isn't any further activity. We therefore assume that the user has lost interest or resolved the problem on their own.

Don't worry though; if this is an error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

**Comment by aNNiMON** [comment]
Still reproducible in 4.10.2

**Comment by aNNiMON** [comment]
Still reproducible in 4.16.6

**Comment by aNNiMON** [comment]
Still reproducible in 5.6.1

**Comment by soredake** [comment]
Not stale.

**Comment by soredake** [comment]
not stale

**Comment by 23rd** [comment]
> ### Expected behaviour
> After deleting pic3, the group picture should be set to pic2
> 
> ###


@aNNiMON  @soredake Hi guys.
Can you tell me where I can find similar behavior? I checked both iOS and Android apps, and after deleting the current group photo, the group photo is completely reset and becomes empty, just like on the desktop.

**Comment by aNNiMON** [comment]
@23rd, correct. However, please also take into account that the Edit Group/Channel screen on Telegram Android only displays a single group photo, so it's expected that removing this photo will clear the group photo entirely. Telegram X Android is even better; it shows a "Remove group photo" or "Remove channel photo" button when the user tries to update the photo.

tdesktop has a different approach. The user needs to open the current photo and delete it. The problem is that previous photos are displayed here as well, so the user thinks that removing the current photo will automatically switch to the next one, if there's one. 

What would be great, at least for me, is to have a separate option in the context menu to remove the group/channel photo. Here:
<img width="419" height="251" alt="Image" src="https://github.com/user-attachments/assets/4d80807b-1b21-43fc-8adb-595f9855ff8f" />

And here:
<img width="290" height="265" alt="Image" src="https://github.com/user-attachments/assets/6fa5a2e0-43ce-4350-a1ca-fe5865344f13" />

**Comment by aNNiMON** [comment]
Still an issue on 6.9.3  
The main issue here is that after removing the group photo, there is no way to preview all previous group photos and set one of them again as the current. You need to upload any group photo again to see the list, then set one photo as main and then delete recently uploaded.
